game/ato/flightplans/airassault.py

```diff
@@ -108,24 +110,17 @@ def layout(self) -> AirAssaultLayout:
             "Dropoff zone",
             self.package.target.position.point_from_heading(heading, 1200),
         )
-        drop_off = builder.cargo_dropoff(drop_off_zone, self.flight.is_helo)
 
         return AirAssaultLayout(
             departure=builder.takeoff(self.flight.departure),
-            nav_to_pickup=builder.nav_path(
-                self.flight.departure.position,
-                pickup_position,
-                altitude,
-                altitude_is_agl,
-            ),
```

Comment:
I personally do like the nav waypoint so that i have the possibility to build kind of an ingress route with it. But maybe it would make more sense to just add a real ingress on the way

Reply:
nvm i just realized it was the nav_to_pickup not dropoff